CONTRIBUTING.md

```diff
@@ -92,6 +93,50 @@ Your PR can contain more than one patch. Use as many patches as necessary to
 implement the request. Each PR should only cover one topic. If you mix up
 different items in your patches or PR, they will likely need to be reworked.
 
+### Assisted PR workflow
+
+If your PR is deemed useful but you are struggling to update it based on
+review feedback, or your PR appears to have been abandoned, the project
+maintainers may suggest (or decide in your absence) that an assisted PR
+workflow be employed. This involves landing your PR and then creating a
+"fixup" PR to resolve any outstanding issues.
```

Comment:
May I suggest a bit different approach? Instead of just merging the original PR, we can create a new PR that takes part/all of the original commits, and add more commits to fix it to a state such that we can merge it like any normal PR.

The original commits should remain to be credited to the original developer by keeping the SOB line in commit message and author in the commit log.

Reply:
Agreed and fixed! :smile: